src/classes/GE_DefaultTemplate.cls

```diff
@@ -0,0 +1,190 @@
+
+        if (!defaultTemplateSRList.isEmpty()) {
+            defaultTemplateSRJSON = defaultTemplateSRList[0].Body.toString();
+        }
+
+        // Get a mapping of partial dev names to full dev names for this org.
+        // Example:
+        // Account1_Name -> Account1_Name_ee63cf9df
+        // 
+        // This is used for updating the base JSON's dev names with the org's dev names. 
+        Map<String, String> updatedDevNameMap = getCurrentDevNameMapping();
+
+        FORM_Template defaultTemplate = (FORM_Template)JSON.deserialize(defaultTemplateSRJSON, FORM_Template.class);
+        FORM_Layout defaultLayout = defaultTemplate.layout;
+        List<FORM_Section> sectionList = defaultLayout.sections;
+
+        for (FORM_Section section : sectionList) {
+            List<FORM_Element> elementList = section.elements;
+
+            for (FORM_Element element : elementList) {
+                if (element.elementType == ELEMENT_TYPE_FIELD) {
+
+                    // Get the updated dev name for this org and replace it into the Base JSON
+                    String partialDevName = element.dataImportFieldMappingDevNames.get(0).substringBeforeLast('_');
+
+                    // If the updated Dev Name map doesn't contain the partial dev name
+                    // it is not necessary to be updated. It is probably a field
+                    // that is on the DI itself and not a mapped field.
+                    // e.g. 'Donation_Donor__c', 'Account1Imported__c', etc
+                    if (updatedDevNameMap.containsKey(partialDevName)) {
+                        element.dataImportFieldMappingDevNames.set(0, updatedDevNameMap.get(partialDevName));
+                    } else {
+                        String fullDevName = element.dataImportFieldMappingDevNames.get(0);
+
+                        if (fullDevName.endsWith('__c')) {
+                            fullDevName = UTIL_Namespace.StrTokenNSPrefix(fullDevName);
+                        }
+
+                        // If it is a DI field, apply the namespace, if there is one. 
+                        element.dataImportFieldMappingDevNames.set(0, fullDevName);
+                    }
+                }
+            }
+        }
+
+        // Update the batch header fields with the namespace prefix, if necessary
+        List<FORM_BatchHeaderField> batchHeaderFieldList = defaultTemplate.batchHeaderFields;
+        
+        for (FORM_BatchHeaderField batchHeaderField : batchHeaderFieldList) {
+
+            if (batchHeaderField.apiName.endsWith('__c')) {
+                batchHeaderField.apiName = UTIL_Namespace.StrTokenNSPrefix(batchHeaderField.apiName);
+            }
+        }
+
+        newDefaultTemplate.Name = defaultTemplate.name;
+        newDefaultTemplate.Description__c = defaultTemplate.description;
+        newDefaultTemplate.Template_JSON__c = JSON.serialize(defaultTemplate);
+        newDefaultTemplate.Format_Version__c = '1.0';
```

Comment:
Let's define a constant for '1.0' in case we'll use a different value in the future.